examples/example-1-themes.html

```diff
@@ -30,7 +30,7 @@
   </div>
 
 <script src="examples/script.js"></script>
-<script src="build/cookieconsent.min.js"></script>
+<script src="src/cookieconsent.js"></script>
```

Comment:
I think referencing src like this will break it, as it gets the CSS from the same directory as whatever you pulled cookieconsent from